### ISSUE TITLE
Diktat codestyle should prohibit empty line before the closing quote

### ISSUE THREAD

**Issue by akuleshov7** [cqfn/diKTat#1054]
```
fun foo() {
    bar()

}
```

the following code should be prohibited

**Comment by petertrr** [comment]
When I run it on diktat-demo with diktat 1.0.0-rc.3, it's fixed or warned about correctrly:
`Warn (3:10) [TOO_MANY_BLANK_LINES] too many consecutive blank lines: do not put newlines at the end of code blocks`
Does it need a more complex reproducer?

**Comment by akuleshov7** [comment]
yes, I have incorrectly understood the bug:. Diktat incorrectly fixes the code:
```
fun foo() {
    bar()


}
```
